agent/bin/ci.sh

```diff
@@ -36,6 +36,8 @@ pub get
 
 dart bin/agent.dart ci -c "$CONFIG_FILE" 2>&1 >> "$LOG_FILE" &
```

Comment:
I think this should be:
```suggestion
dart bin/agent.dart ci -c "$CONFIG_FILE" >> "$LOG_FILE" 2>&1 &
```
In order to redirect the error output that you want.